packages/main/src/components/VariantManagement/index.tsx

```diff
@@ -318,7 +318,8 @@ const VariantManagement = forwardRef<HTMLDivElement, VariantManagementPropTypes>
   }, [safeChildrenWithFavorites]);
 
   // todo: this applies if `readOnly` is set to `false` as well since the value is read via data-attribute and is therefore a string not a boolean
```

Comment:
btw should this be a `todo`? it seems to me more like a note explaining some context

Reply:
The comment can be removed as it's not applicable anymore.